### PR TITLE
Adding initial cloud Structure docs

### PR DESCRIPTION
**Pull request by zachgiordano**
- [X] I have read and agree to the contributing guidelines for [submitting new pull requests](https://github.com/griptape-ai/griptape?tab=readme-ov-file#submitting-pull-requests).

## Describe your changes

Adding some initial documentation for Griptape Cloud Structures.

## Issue ticket number and link

----
📚 Documentation preview 📚: https://griptape--1036.org.readthedocs.build//1036/

### CODE REVIEW

**Review by vachillo — DISMISSED** [review]

File: docs/griptape-cloud/index.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Griptape Cloud
 
-Griptape Cloud is a managed platform for running AI-powered agents, pipelines, and workflows.
\ No newline at end of file
+Managed services for your AI app stack. Deploy and scale end-to-end solutions, from LLM-powered data prep and retrieval to AI agents, pipelines and workflows.
```
Comment (collindutter) [inline docs/griptape-cloud/index.md:3]: This reads a bit odd, I think it should still start with "Griptape Cloud is...

File: docs/griptape-cloud/structures/structure-config.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+## Structure Config File
```
Comment (collindutter) [inline docs/griptape-cloud/structures/structure-config.md:1]: Should start with `## Overview` for consistency with other pages.

**Review by collindutter — CHANGES_REQUESTED** [review]

File: docs/griptape-cloud/structures/structure-config.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      - scripts/my-post-build-install-script.sh
+run:
+  main_file: structure.py
+```
+
+### Configuration Fields
+
+#### version
+
+The Structure Config YAML version number.
```
[the remark this answers is not in the document]
Comment (zachgiordano) [reply docs/griptape-cloud/structures/structure-config.md:29]: I can update this to say "schema" version

File: docs/griptape-cloud/structures/structure-config.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+#### runtime_version
+
+The specific version of the runtime environment for the Structure.
+
+#### build
+
+The build-time configuration for the Structure. This configuration is optional.
+
+* **pre_build_install_script** - The path to your pre_build_install_script, for running during the Structure build prior to dependency installation.
```
[the remark this answers is not in the document]
Comment (zachgiordano) [reply docs/griptape-cloud/structures/structure-config.md:43]: yes, I'll update to note that

File: docs/griptape-cloud/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (zachgiordano) [reply docs/griptape-cloud/index.md:3]: I'll link to the managed template and samples. I imagine we'll want to do a rewrite of this main page, so this is just a placeholder.

**Review by collindutter — DISMISSED** [review]

**Review by vachillo — DISMISSED** [review]

**Pull request closed without merging**